UserAgeCalculation

```diff
@@ -9,22 +9,52 @@ namespace UserAgeCalculation
             Console.Write("Enter your name: ");
             string userName = Console.ReadLine();
 
+            int currentYear = DateTime.Now.Year;
 
-            Console.Write("Enter your year of birth: ");
-            int yearOfBirth = Convert.ToInt32(Console.ReadLine());
+            int yearOfBirth;
 
-            int currentYear = DateTime.Now.Year;
-            int userAge = currentYear - yearOfBirth; 
+            while (true)
```

Comment:
this may cause endless loop. I recommend to use int.TryParse inside while. For example: 

string yearOfBirthString = Console.ReadLine();

while(!int.TryParse(yearOfBirthString, out yearOfBirth))
{
     //add here exception message output and getting yearOfBirthString again
     //and while you don't get TryParse result true user will have to reenter year of birth.